### PR TITLE
[20825] Bump version to v2.6.8

### PR DESCRIPTION
**Pull request by Mario-DL**
## Description

This PR bumps Fast DDS docs version to `v2.6.8`

## Contributor Checklist

- [X] Commit messages follow the project guidelines. 
- [x] Code snippets related to the added documentation have been provided.
- [x] Documentation tests pass locally.
- **N/A** Applicable backports have been included in the description.

## Reviewer Checklist

- [x] The PR has a milestone assigned.
- [x] The title and description correctly express the PR's purpose.
- [x] Check contributor checklist is correct.
- [x] CI passes without warnings or errors.

### CODE REVIEW

**Comment by Mario-DL** [comment]
@richiprosima please test docs

**Comment by Mario-DL** [comment]
@richiprosima please test docs

**Comment by Mario-DL** [comment]
@richiprosima please test docs

File: docs/notes/notes.rst [hunk 1]
```diff
@@ -3,64 +3,65 @@
...
-1. Support ``Autofill port`` (:ref:`automatically set a port<transport_tcp_transportDescriptor>`) for TCP Transport.
-2. Define a :ref:`super client<env_vars_ros_super_client>` by environment variable
-3. Support :ref:`TCP Discovery server<use-case-tcp-discovery-server>` CLI and environment variable
-4. Define methods (:ref:`environment variable<env_vars_builtin_transports>`,
-   :ref:`rtps layer<rtps_layer_builtin_transports>`, :ref:`xml<RTPS>`) to
-   :ref:`configure transport scenarios<transport_tcp_enabling>`
-5. :ref:`Secure discovery server<DS_security>`
+#. :ref:`Authentication Handshake Properties <property_policies_security>` documentation.
+#. TCP Client and Server Participant Decision Making.
+#. Add macOS and Ubuntu Github CI.
```
Comment (JesusPoderoso) [inline docs/notes/notes.rst:13]: I would set this features as GitHub CI improvement

File: docs/notes/notes.rst [hunk 2]
```diff
@@ -3,64 +3,65 @@
...
-5. Rerun failed tests with ctest option instead of colcon's.
-6. Several improvements on CI jobs.
-7. Upgrade CMake minimum requirement to 3.16.3
-8. Update PR checklist template. Backports and Description
+#. Make DataWriters always send the key hash on keyed topics.
+#. Include variety of terminate process signals handler in discovery server.
+#. Pin CMake version and ``vm.mmap_rnd_bits`` in sanitizer workflows.
+#. Effectively assert ``AUTOMATIC/MANUAL_BY_PARTICIPANT`` liveliness.
+#. Pick smallest available participant ID for new participants.
+#. Build Fast DDS Python bindings in Fast DDS Docs Github CI job.
```
Comment (JesusPoderoso) [inline docs/notes/notes.rst:22]: This would be a GitHub CI improvement, as well as the one shapes demo related

**Review by JesusPoderoso — CHANGES_REQUESTED** [review]

**Review by JesusPoderoso — APPROVED** [review]
LGTM

**Pull request merged**